src/SDKs/Advisor/Management.Advisor/Microsoft.Azure.Management.Advisor.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<Description>Microsoft Azure Advisor Library</Description>
+		<AssemblyName>Microsoft.Azure.Management.Advisor</AssemblyName>
+		<Version>0.9.0-preview</Version>
+		<PackageId>Microsoft.Azure.Management.Advisor</PackageId>
+		<PackageTags>Management.Advisor;Advisor;recommendations;</PackageTags>
+		<PackageReleaseNotes>See https://aka.ms/azure-sdk-for-net/advisor/changelog for release notes.</PackageReleaseNotes>
```

Comment:
Could you check this link? Or you could just specify the notes, an example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L12)